root/static/scripts/edit/externalLinks.js

```diff
@@ -615,6 +616,13 @@ export class ExternalLinksEditor
           error.message = l(`This URL is not allowed 
                     for the selected link type.`);
         }
+        if (error.target ===
+          URLCleanup.ERROR_TARGETS.ENTITY) {
+          error.message = exp.l(
+            `This URL is not allowed for “{entity_type_name}” entities.`,
+            {entity_type_name: formatEntityTypeName(checker.entityType)},
```

Comment:
I think a separate string for each entity type (like we do for SEARCH_PLACEHOLDERS, "Add a new x," "Add another x," etc. would be nice here; saying `“artist” entities` instead of `artists` just looks and sounds awkward. Like `Yesterday I watched a movie of the genre “horror”` or something. :laughing:

Reply:
That is fair, it shouldn't be a crazy amount of strings and they should be easy to translate anyway :)

Reply:
Thanks, I was just too lazy here. Split in the commit <https://github.com/metabrainz/musicbrainz-server/pull/2280/commits/cb27c59e6d85b62f1ef77aa4129f637c6f9fd1e6>.